### PR TITLE
start the cleanup

### PR DESCRIPTION
**Pull request by linusnorton**
Clean up the gradle file

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/divorce/casemaintenanceservice/config/HttpConnectionConfiguration.java [hunk 1]
```diff
@@ -86,7 +85,6 @@ private ClientHttpRequestFactory getClientHttpRequestFactory(
         CloseableHttpClient client = HttpClientBuilder
             .create()
             .useSystemProperties()
-            .addInterceptorFirst(new OutboundRequestIdSettingInterceptor())
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/reform/divorce/casemaintenanceservice/config/HttpConnectionConfiguration.java:89]: I thought we need this for logging request id by the java logging library https://github.com/hmcts/java-logging/blob/master/java-logging-httpcomponents/src/main/java/uk/gov/hmcts/reform/logging/httpcomponents/OutboundRequestIdSettingInterceptor.java#L25 🤔

Comment (linusnorton) [reply src/main/java/uk/gov/hmcts/reform/divorce/casemaintenanceservice/config/HttpConnectionConfiguration.java:89]: I checked with the tech leads and Tim said it was before AppInsights. There was a custom elk logging stack

**Review by nitinprabhu — APPROVED** [review]
🚒

File: dependency-check-suppressions.xml [hunk 2]
```diff
@@ -1,130 +1,29 @@
...
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
-        <cve>CVE-2020-17527</cve>
-        <cpe>cpe:/a:java-websocket_project:java-websocket</cpe>
-        <cpe>cpe:/a:apache_tomcat:apache_tomcat</cpe>
-        <cpe>cpe:/a:apache_software_foundation:tomcat</cpe>
-        <cpe>cpe:/a:apache:tomcat</cpe>
-    </suppress>
-    <suppress>
-        <cve>CVE-2020-8908</cve>
```
Comment (velimd) [inline dependency-check-suppressions.xml:128]: wouldnt removing all the checks break something (code wise)?

File: build.gradle [hunk 3]
```diff
@@ -421,6 +270,8 @@ task nightlyFunctional(type: Test, description: 'Runs the functional tests.', gr
 
             systemProperty 'AUTH_PROVIDER_SERVICE_CLIENT_KEY', getVaultSecret('cms-service-key')
             systemProperty 'AUTH2_CLIENT_SECRET', getVaultSecret('idam-secret')
+            // systemProperty 'IDAM_API_BASEURL', 'http://localhost:5000'
+            // systemProperty 'CASE_DATA_STORE_BASEURL', 'http://localhost:4452'
```
Comment (mapingo) [inline build.gradle:268]: Do these commented lines need to stay?  Can they be deleted?

File: build.gradle [hunk 4]
```diff
@@ -461,6 +312,20 @@ pitest {
     mutationThreshold = 80
 }
 
+// https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/configuration.html
```
Comment (mapingo) [inline build.gradle]: Does the commented line need to stay?

File: dependency-check-suppressions.xml (hunk 2 above)
Comment (linusnorton) [reply dependency-check-suppressions.xml:128]: Many no longer apply. I've removed them all and re-added the ones that are relevant

File: build.gradle (hunk 3 above)
Comment (linusnorton) [reply build.gradle:268]: For now, I will do a separate PR to change the way we toggle between testing locally vs testing in AAT

File: build.gradle (hunk 4 above)
Comment (linusnorton) [reply build.gradle]: No that one can go

**Review by mapingo — APPROVED** [review]

**Pull request merged**